src/index.html

```diff
@@ -6,9 +6,62 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="menu">
+        <a href="index.html" class="logo">
+
+          <img
+            src="images/logo.png"
+            alt="Moyo logo"
+            class="header__logo"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#"
+              data-qa="hover" class="nav__link laptops">Laptops & Computers</a>
```

Comment:
```suggestion
              <a 
                href="#"
                data-qa="hover" 
                class="nav__link laptops"
              >
                Laptops & Computers
              </a>
```

Reply:
<img width="1125" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/d5a1db45-805a-4a8a-9da4-541497a1388b">
fix it everywhere